front/pages/api/w/[wId]/apps/[aId]/runs/index.ts

```diff
@@ -192,6 +193,16 @@ async function handler(
           });
         }
 
+        logger.info(
+          {
+            owner: owner.sId,
+            targetOwner: targetOwner.sId,
+            user: user.sId,
+            app: app.sId,
+          },
+          "wIdTarget access"
```

Comment:
Nit:
I’d go with a more explicit log message it might make sense as we are into it but in a few months time from now pretty hard to understand what this log means at first glance.

Reply:
Motivation for short logs is that it makes them easier to search in DD :-)

This log is not intended for debugging but rather monitoring and it's call site gives total context no?

Reply:
Right now it does not sure in 6 months 🙃